docs_src/microservices/device/supported/device-onvif-camera/Walkthrough/setup.md

```diff
@@ -105,6 +105,41 @@ Clone the EdgeX compose repository:
 git clone https://github.com/edgexfoundry/edgex-compose.git
 ```
 
+## Proxy Setup (Optional)
+
+!!! Note
+    The device used for deployment of device-onvif-service must be behind proxy/VPN.
+
+
+Setup Docker Daemon or Docker Desktop to use proxied environment.
+
+- Follow guide [here](https://docs.docker.com/config/daemon/systemd/#httphttps-proxy) for Docker Daemon proxy setup
+
+- Follow guide [here](https://docs.docker.com/desktop/settings/windows/#proxies) for Docker Desktop proxy setup
+
+!!! example - "http.conf file to configure Docker Client proxy"
+    ```
+        {
+            "proxies": {
+                "httpProxy": "http://proxy.example.com:3128",
+                "httpsProxy": "https://proxy.example.com:3129",
+                "noProxy": "*.test.example.com,.example.org,127.0.0.0/8"
+            }
+        }
+    ```
+
+!!! Note - "Note if building custom images"
+    If building your own custom images, set environment variables for HTTP_PROXY, HTTPS_PROXY and NO_PROXY
```

Comment:
Do we need to either amend our Makefile or instruct the user how to add the `--build_arg` arguments?

Reply:
Yes, the device-usb-service already has the proxy build-args in the build script. Just need to add the build-args to the device-onvif-service... I can make that addition as a follow-up PR for the device-onvif-service.